pghoard/pghoard.py

```diff
@@ -274,13 +277,47 @@ def delete_remote_wal_before(self, wal_segment, site, pg_version):
                 self.log.exception("Problem deleting: %r", wal_path)
                 self.metrics.unexpected_exception(ex, where="delete_remote_wal_before")
 
-    def delete_remote_basebackup(self, site, basebackup, metadata):
+    def _get_delta_basebackup_files(self, site, storage, metadata, basebackup, keep_backups):
```

Comment:
Naming could be improved to make it clearer `basebackup` is a basebackup to delete.